samples/client/petstore/python-asyncio/tests/test_pet_api.py

```diff
@@ -27,15 +27,6 @@
 HOST = 'http://localhost:80/v2'
 
 
-def async_test(f):
-    def wrapper(*args, **kwargs):
-        coro = asyncio.coroutine(f)
-        future = coro(*args, **kwargs)
-        loop = asyncio.get_event_loop()
-        loop.run_until_complete(future)
-    return wrapper
```

Comment:
I only see this being used in this one file. Why did you move it to the util file?
If it is used in more than one file it makes sense to put it there as a common utility tool, but if it is only needed here can you keep it here?

Reply:
Yeah, my mistake... I've just added new file.

Reply:
Looks good as-is